src/integrationTest/resources/logback-test.xml

```diff
@@ -7,7 +7,6 @@
         </encoder>
     </appender>
     <logger name="org.hibernate" level="warn" additivity="true"/>
-    <logger name="org.hibernate.engine.jdbc.env.internal.JdbcEnvironmentInitiator" level="debug" additivity="true"/>
```

Comment:
We don't need to have the `debug` level enabled to log the exception we throw from `MongoDialect.checkVersion`. I believe, previously the `debug` level was needed.